OSS_Guidance/Data_Management_Sharing.md

```diff
@@ -23,7 +23,7 @@ Scientific data resulting from SMD-funded scientific activities shall be made pu
 2. Any scientifically useful data associated with a SMD research award shall be made publicly available by the end of the period of performance, whether or not the data would be needed to validate the scientific conclusions of a peer-reviewed publication. 
 
 ## Where to Share Data 
-Data must be shared and archived in locations that ensure the accessibility and preservation of the data. Repositories should meet the guidelines for SMD-acceptable data repositories in Appendix D of [SPD-41a](https://science.nasa.gov/science-red/s3fs-public/atoms/files/SMD-information-policy-SPD-41a.pdf), which are based upon the [Desirable Characteristics of Data Repositories for Federally Funded Research](https://www.whitehouse.gov/wp-content/uploads/2022/05/05-2022-Desirable-Characteristics-of-Data-Repositories.pdf). The method for sharing the data must be described as part of the data management plan.    
+Data must be shared and archived in locations that ensure the accessibility and preservation of the data. Repositories should meet the guidelines for SMD-acceptable data repositories in Appendix D of [SPD-41a](https://smd-cms.nasa.gov/wp-content/uploads/2023/08/smd-information-policy-spd-41a.pdf), which are based upon the [Desirable Characteristics of Data Repositories for Federally Funded Research](https://www.whitehouse.gov/wp-content/uploads/2022/05/05-2022-Desirable-Characteristics-of-Data-Repositories.pdf). The method for sharing the data must be described as part of the data management plan.    
```

Comment:
It would be better to use the "evergreen" link for SPD-41 of https://science.nasa.gov/spd-41/

Reply:
This is done.